routes/apiRoutes.js

```diff
@@ -1,149 +1,81 @@
-  res.send('Welcome to the West Coast Hurricane API!');
-});
+const router = express.Router();
 
+/* Date Endpoint */
 
-router.get('/hurricanes', async (req, res) => {
+router.get('/date', async(req,res) => {
   try {
-    const hurricanes = await db.hurricane.findAll();
-    const reply = hurricanes.length > 0 ? { data: hurricanes } : { message: 'no results found' };
-    res.json(reply);
+    console.log('touched /date with GET');
+    const result = await db.sequelizeDB.query(date.getDate, {
+      type:sequelize.QueryTypes.SELECT
+    });
+    res.json(result);
   } catch (err) {
-    console.error(err);
-    res.error('Server error');
+      console.log(error);
   }
-});
-
-export default router;
-/* City Endpoint */
-import endpoint1 from '../routes/cityControllers'
-
-router.route('/city')
-  .get(async (req, res) => {
-    try {
-      console.log('touched /city with GET');
-      res.json({ message: 'GET endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .put(async (req, res) => {
-    try {
-      console.log('touched /city with PUT');
-      res.json({ message: 'PUT endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .post(async (req, res) => {
-    try {
-      console.log('touched /city with POST');
-      res.json({ message: 'POST endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .delete(async (req, res) => {
-    try {
-      console.log('touched /city with DELETE');
-      res.json({ message: 'DELETE endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-
-    router.get('/', (req, res) => {
-      res.json('You have touched the city endpoint');
-    });
-  });
-
-/* Date Endpoint */
-
-router.route('/date')
-  .get(async (req, res) => {
-    try {
-      console.log('touched /date with GET');
-      res.json( {message: 'GET endpoint'} );
-    } catch (err) {
-      console.log(err);
-    }      
-  })
-
-  .put(async (req, res) => {
+})
+
+// router.get('/date', async (req, res) => {
+//     try {
+//       console.log('touched /date with GET');
+//       if (!req.body) {
+//         const result = await db.sequelizeDB.query(date.getDate, {
+//           replacements: {
+//             id: 'req.body.id'
+//           },
+//           type: sequelize.QueryTypes.SELECT
+//         })
+//       }
+//       else {
+//         const result = await db.sequelizeDB.query(date.getDateByID, {
+//           replacements: {
+//             id: req.body.id }
+//         })
+//       }
+//       res.json(result);
+//     } catch (err) {
+//       console.log(err);
+//     }      
+//   })
+
+router.put(async (req, res) => {
     try {
       console.log('touched /date with PUT');
-      res.json( {message: 'PUT endpoint'} );
+      const result = await db.sequelizeDB.query(date.putDate, {
+        replacements: {
+          id: req.body.id, date: req.body.date
+        },
+        type: sequelize.QueryTypes.UPDATE
+      });
```

Comment:
Busted here because there's no response being sent to close the `try`, needs a `res.json({message: result})`